Caltech-IPAC/firefly
# firefly-468-Unit test is needed for Wavelength calculation for different WCS

https://jira.ipac.caltech.edu/browse/FIREFLY-468

 1. Add utility methods to load table from a FITS file
  2. Add utility methods to read binary table in the FITS
  3. Add utility methods to create json files that contains FITs header and the binary table information
  4. Create testing data for wavelength unit test
  5. Add unit tests for Wavelength.js
  NOTE:  The unit test for each algorithm is black box type unit test.  The expected value is calculated either by directly taking from the known data or calculated independently using the known data.  

To test:
 1. check out the data branch firefly-468-wavelength-unitTest in firefly_test_data
 2. check out this branch
 3. gradle :fi:jTest
